<!doctype html>
<html>
<head>
    <meta charset="gb2312">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <meta name="product" content="metro ui css framework">
    <meta name="description" content="simple responsive css framework">
    <meta name="author" content="sergey s. pimenov, ukraine, kiev">

    <link href="css/metro-bootstrap.css" rel="stylesheet">
    <!-- <link href="css/metro-bootstrap-responsive.css" rel="stylesheet"> -->
    <link href="css/iconfont.css" rel="stylesheet">
    <link rel="stylesheet" type="text/css" media="screen,print" href="css/mypublications.css" />
    

    <!-- load javascript libraries -->
    <script src="js/jquery/jquery.min.js"></script>
    <script src="js/jquery/jquery.widget.min.js"></script>

    <!-- metro ui css javascript plugins -->
    <script src="js/load-metro.js"></script>

    <!-- local javascript -->
    <script src="js/github.info.js"></script>

    <title>Synthetic Biology</title>
    <script src="js/metro.min.js"></script>
    <style>
        .container {
            width: 1150px;
        }
    </style>
</head>
<body class="metro" style="margin:0 auto;width: 1200px">
    <div class="container">
        <header class="margin20 nrm nlm">
            <div class="clearfix">
                <a class="place-left" href="#" title="">  
                    <h1><font face='Helvetica'>Tsinghua Dai Lab</font></h1>
                </a>
                <a class="" href='http://www.tsinghua.edu.cn/publish/newthu/index.html' style="margin-left:200px">
                    <img src="images/Tsinghua.png"  />
                </a>
                <a class="" href='http://life.tsinghua.edu.cn' style="margin-left: 30px">
                    <img src="images/logo.gif" />
                </a>
            </div>

            <div class="main-menu-wrapper">
                <ul class="horizontal-menu nlm">
                    <li><a href="DaiLab.html">Dai Lab</a></li>
                    <li><a href="LabMembers.html">Lab Members</a></li>
                    <li><a href="Publications.html">Publications</a></li>
                    <li><a href="Lab Activities.html">Lab Activities</a></li>
                    <li><a href="Jobs.html">Jobs</a></li>
                    <li>
                    <a class="dropdown-toggle" href="#">Research</a>
                    <ul class="dropdown-menu" data-role="dropdown">
                        <li><a href="Chromatin Biology.html">Chromatin Biology</a></li>
                        <li><a href="Synthetic Biology.html">Synthetic Biology</a></li>
                        <li><a href="Algal Biology.html">Algal Biology</a></li>
                    </ul>
                    </li>
                </ul>
            </div>
        </header>

        <div class="main-content clearfix">
            <div class="tile-area no-padding clearfix">
                <div class="tile-group no-margin no-padding clearfix" style="width: 100%">
                    <div class="tile onedouble quadro-vertical bg-gray ol-transparent" style="float: right; ">
                        <div class="tile-content">
                            <div class="brand">
                                <span class="label fg-white"></span>
                            </div>
                        </div>
                    </div>
                    <div class="body-text" style="width: 900px">
                        <div class="paragraph_style">
                            Research: Chromatin Biology
                        </div>
                        <h3>We will combine genetics, biochemistry and other tools to study the function of chromatin.</h3>
                        <p> &nbsp &nbsp In the past decades, histones, once considered by many as ¡°The World¡¯s Most Boring Protein¡± has rejuvenated, thanking for the discovery of various covalent modifications. A ¡°histone code¡± was hypothesized that specific pattern of modifications on histones can be ¡°read¡± and ¡°written¡± as a code by protein executors to carry out specific biological function. In addition to the individual and combinatorial effects of different modifications, another layer of complexity is created by how these histone markers are regulated temporally and spatially, which has not been fully explored. Furthermore, the journey to understand the function of histone modifications including their modifying enzymes in certain biological processes such as meiosis, development, tumorigenesis and aging just gets started. On the other hand, another unsolved question in the filed is how many markers occur on a given histone or nucleosome simultaneously and how could they affect higher-order chromatin structure/organization. We want to study the following aspects on chromatin biology:</p><br />
                        <ol>
                            <li>Identify new modifications on histones and other chromatin associated proteins and dissect their biological function. We recently identified a new class of modifications on histones and are working actively on identifying its modifying enzymes and analyzing the biological significance of this modification. </li>
                            <li>Continue our systematic mutagenesis study on histone H2A and histone H2B. We will use this new library and our histone H3 and histone H4 mutant library as a versatile tool kit to address questions such as a) how histone H2A and histone H2B function in DNA-damage response? Where are the critical residues/regions? b) Which region in nucleosome core particle is important for maintaining genome stability? c) Are there any synthetic lethal interactions between H3/H4 and H2A/H2B mutants? d) In collaboration with Nevan Krogan¡¯s lab at UCSF, we will analyze both H3/H4 and H2A/H2B mutants using their E-MAP technology and eventually test synthetic interaction between the histone mutants and all yeast genes.</li>
                            <li>Study the dynamics of histone modifications during multiple cellular processes. We will focus on three major cellular processes: a) Meiosis and mating process. Conceptually, yeast meiosis/sporulation is similar to spermatogenesis in higher eukaryote and both require remodeling and compacting the genome and therefore, can serve as a good model to study spermatogenesis. In fact the yeast might be the best system to study meiosis since the four spores can be easily segregated to study the phenotype for each progeny. Using the mutant libraries we have for both H3/H4 and H2A/H2B, we will perform a complete survey of every residue in core histones for their function during different stages of sporulation. In addition, we will analyze the chromatin changes during mating process and how those changes are affected by different mutations. b) Cell cycle. We will develop a Mass Spectrometry methodology to identify histone modifications both qualitatively and quantitatively during the process of cell cycle. We will purify histone samples at different stage during the cell cycle and identify/quantify specific modifications. c) Aging process. We will utilize the ¡°molecular barcode¡± incorporated into each histone mutant during synthesis to screen histone mutant strains as a pool to identify mutations with altered chronological and/or replicative life span. We will determine the mutant population at different time points by microarray or high-throughput sequencing. </li>
                            <li>Develop a Drosophila system to study histone mutagenesis. We will create a conditional knockout strain for the whole histone locus (luckily, all histones-coding genes are located at the same locus in fly genome) in collaboration with Guanjun Gao¡¯s lab here at Tsinghua University. We will integrate the mutated histones into the same chromosome location and study their function.</li>
                           </ol>
                       </div>
                   </div>
               </div>
           </div>
       </div>
       </body>
       </html>



